docs/DeveloperGuide.md

```diff
@@ -171,6 +171,46 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Tag Management
+
+This section will describe tag management in the address book as well as the features implemented.
+
+### Centralising Tags in the Address Book
+
+In the previous implementation, all `Tag`s are independent of one another despite having the same tag names. Hence, centralising the tags enable the user to easily manipulate them as well as easily searching for `Person`s that contain the tag.
```

Comment:
> Hence, centralising ...

There is nothing to continue from, `hence` does not make sense in this context.

Reply:
Rephrased in 61207721